pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -285,3 +261,17 @@ func (o *objectWatcherImpl) NeedsUpdate(clusterName string, desiredObj, clusterO
 
 	return lifted.ObjectNeedsUpdate(desiredObj, clusterObj, version), nil
 }
+
+func (o *objectWatcherImpl) validateUpdatePermission(clusterName string, desiredObj, clusterObj *unstructured.Unstructured) error {
+	if util.GetLabelValue(desiredObj.GetLabels(), workv1alpha1.WorkNameLabel) != util.GetLabelValue(clusterObj.GetLabels(), workv1alpha1.WorkNameLabel) {
+		conflictResolution := util.GetAnnotationValue(desiredObj.GetAnnotations(), workv1alpha2.ResourceConflictResolutionAnnotation)
+		if conflictResolution != workv1alpha2.ResourceConflictResolutionOverwrite {
+			// The existing resource is not managed by Karmada, and no conflict resolution found, avoid updating the existing resource by default.
+			return fmt.Errorf("resource(kind=%s, %s/%s) already exist in cluster %v and the %s strategy value is empty, karmada will not manage this resource",
```

Comment:
When a resource conflict occurs, the error log can be directly printed, but no error is returned. In this case, the resource will not be updated repeatedly.